src/customer/strategies/googlepay/googlepay-customer-strategy.ts

```diff
@@ -95,33 +99,31 @@ export default class GooglePayCustomerStrategy implements CustomerStrategy {
         throw new InvalidArgumentError();
     }
 
-    private _onPaymentSelectComplete(): void {
-        this._formPoster.postForm('/checkout.php', {
-            headers: {
-                Accept: 'text/html',
-                'Content-Type': 'application/x-www-form-urlencoded',
-            },
-        });
-    }
-
-    private _onError(error?: Error): void {
-        if (error && error.message !== 'CANCELED') {
-            throw error;
-        }
-    }
-
     @bind
     private _handleWalletButtonClick(event: Event): Promise<void> {
         event.preventDefault();
 
         return this._googlePayPaymentProcessor.displayWallet()
             .then(paymentData => this._googlePayPaymentProcessor.handleSuccess(paymentData)
-            .then(() => {
-                if (paymentData.shippingAddress) {
-                    this._googlePayPaymentProcessor.updateShippingAddress(paymentData.shippingAddress);
-                }
-            }))
+                .then(() => {
```

Comment:
same as I commented in other file, maybe close the brackets earlier to avoid extra indentation or switch to async/await. Thanks!